kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -257,4 +286,67 @@
     margin-left: 0.75em;
   }
 
+  @media screen and (max-width: 600px) {
+    .sync-status {
+      max-width: 400px;
+
+      .a {
+        margin-right: 13px;
+      }
+
+      .disco {
+        margin-right: -640px;
+      }
+
+      span {
+        margin-left: -191px;
+        word-wrap: break-word;
+      }
+    }
+
+    .wifi-svg {
+      margin-left: -213px;
+    }
+  }
+  @media screen and (min-width: 600px) and (max-width: 1100px) {
+    .sync-status {
+      .a {
+        margin-right: 13px;
+      }
+
+      .disco {
+        margin-right: -640px;
```

Comment:
I noticed more than 10 instances of negative margins being used. While they can be useful at times, they can be difficult to debug and make the CSS harder to read, so we try to avoid them or add a comment explaining its usage. Is there an alternative you can use in place of some of these?